libs/services/designer-client-services/src/lib/consumption/operations/functions.ts

```diff
@@ -15,19 +15,21 @@ export const functionGroup = {
   },
 };
 
+const api = {
+  id: '/connectionProviders/azureFunctionOperation',
```

Comment:
Get these from operationgroup.ts in this folder so we have only one place to define the connector details. See how it is done for xslt and other IA operations